src/shared/model/Errors.ts

```diff
@@ -8,6 +8,7 @@ export enum IdapiErrorMessages {
   USER_ALREADY_VALIDATED = 'User Already Validated',
   BREACHED_PASSWORD = 'Breached password',
   EMAIL_IN_USE = 'Email in use',
+  INVALID_EMAIL_ADDRESS = 'Invalid emailAddress:',
```

Comment:
Handle this error which is returned by IDAPI if an invalid/empty email address is `POST`ed